src/eventsRelay/ContractEventsSubscription.js

```diff
@@ -27,11 +27,11 @@ async function getPastEvents(contract, fromBlock, toBlock) {
 }
 
 module.exports = class ContractEventsSubscription extends EventEmitter {
-  constructor(web3, contract, fromBlock = null) {
+  constructor(web3, contracts, fromBlock = null) {
```

Comment:
Why just not create 3 subscriptions?

Reply:
Because I was not sure where they get used across the code. Seemed like a nice way to isolate the changes.

Reply:
Let's not touch «primitive» stuff if we can

Reply:
In latest commit i changed the name to ContractsEventSubscribtion to make more sense. The old object is just a special case where the contracts list has only 1 element in.

Reply:
You basically did opposite thing :-) So if i'll want to subscribe for events to another contract, i need to do what? Add more logic into subscription class?

If you have a function `add(n1, n1)` you not need to change it if you need to add 3 numbers

Reply:
I am not sure what you are trying so say. My point is, if I have a contract C, calling ContractEventsSubsribtion(C) or calling ContractsEventSubsribtion([C]) will give me an object that behaves EXACTLY the same way i.e. they are identical. The only difference is that now I have an option to call call ContractsEventSubsribtion([C, A, B]) and listen to events from all 3 contracts through the same object.

Reply:
You might be worried because I changed the interface of this object, but I made sure to change all the calls to it also.

Reply:
Ok